### PR TITLE
Issue438 update db tenant

### LINKED ISSUE #438: Update db for Tenant Model
**Purpose**
The following fields should be required:
`firstName`
`lastName`
`phone`

**Proposed Solution**
add `nullable=False` to the three fields

### PR DESCRIPTION
**Pull request by aedwardg**
### What issue is this solving?

Closes codeforpdx/dwellingly-app/issues/438

- Ensures `firstName`, `lastName` and `phone` fields are required for a new Tenant.
- Validates existence and length of those three fields
- Tests validations with pytest

### Any helpful knowledge/context for the reviewer?
Is a re-seeding of the database necessary? YES
Any new dependencies to install? NO
Any special requirements to test? NO

### Please make sure you've attempted to meet the following coding standards
- [x] Code builds successfully
- [x] Code has been tested and does not produce errors
- [x] Code is readable and formatted
- [x] There isn't any unnecessary commented-out code

### CODE REVIEW

File: schemas/tenant.py [hunk 1]
```diff
@@ -1,13 +1,30 @@
...
 
-  addedOn = fields.DateTime(time_format)
-  created_at = fields.DateTime(time_format)
-  updated_at = fields.DateTime(time_format)
+    created_at = fields.DateTime(time_format)
+    updated_at = fields.DateTime(time_format)
+
+    firstName = fields.Str(
+        required=True,
+        validate=validate.Length(min=1, max=100),
```
Comment (NickSchimek) [inline schemas/tenant.py:16]: I'm not familiar with this method. It seems assigning it to validate has some automatic message handling happening? Is that correct?

**Review by NickSchimek — APPROVED** [review]
Thanks for the PR. This looks really good. 💫

File: schemas/tenant.py (hunk 1 above)
Comment (aedwardg) [reply schemas/tenant.py:16]: Yes, I just used one of the built-in validators for this:
https://marshmallow.readthedocs.io/en/stable/marshmallow.validate.html#api-validators

Looking at the source code, each one has built-in message(s) if you don't provide the optional `error` argument:
https://marshmallow.readthedocs.io/en/stable/_modules/marshmallow/validate.html#Length

Comment (NickSchimek) [reply schemas/tenant.py:16]: good to know! Thank you.

**Pull request merged**